README.md

```diff
@@ -93,6 +128,22 @@ If a build fails, SCons will usually create a file called *config.log*.  If you
 look towards the end of the file you can often see exactly what failed.  When
 reporting build problems it is a good idea to include this file in the report.
 
+### C! and 64-bit Windows
+C! targets 32-bit Windows by default. You can confirm by checking the start of
```

Comment:
I don't actually know if this is the case, but it's the behaviour I observed with the current master of https://github.com/CauldronDevelopmentLLC/cbang